ert_shared/services/_storage_main.py

```diff
@@ -179,7 +179,41 @@ def terminate_on_parent_death():
 
 if __name__ == "__main__":
     # pylint: disable=W0611
-    import ert_logging
+    import logging
+    import logging.config
+
+    logging_conf = {
+        "version": 1,
+        "disable_existing_loggers": True,
+        "formatters": {
+            "standard": {"format": "%(asctime)s [%(levelname)s] %(name)s: %(message)s"},
+        },
+        "handlers": {
+            "default": {
+                "level": "WARNING",
+                "formatter": "standard",
+                "class": "logging.StreamHandler",
+                "stream": "ext://sys.stdout",
+            },
+            "file": {
+                "formatter": "standard",
+                "class": "webviz_ert.logging.TimestampedFileHandler",
+                "level": "DEBUG",
+                "filename": "api-log.txt",
+            },
+        },
+        "loggers": {
+            "": {"handlers": ["default", "file"], "level": "INFO"},
+            "uvicorn.error": {"level": "INFO"},
+            "uvicorn.access": {"level": "INFO"},
+            "ert_shared.storage": {"level": "INFO"},
+            "ert_shared.status": {"level": "INFO"},
+            "res": {"level": "INFO"},
+        },
+    }
+    uvicorn.config.LOGGING_CONFIG.update(logging_conf)
+
+    logging.config.dictConfig(logging_conf)
```

Comment:
Are you sure this is what you want? Seems to me this would achieve the same:
```suggestion
    set_abort_handler(_log_util_abort)
    with open(LOGGING_CONFIG, encoding="utf-8") as conf_file:
        logging_conf = (yaml.safe_load(conf_file))
    uvicorn.config.LOGGING_CONFIG = logging_conf
    logging.config.dictConfig(logging_conf)
```
 then this for the context:
```python
    with ErtPluginContext() as context:
        context.plugin_manager.add_logging_handle_to_root(logging.getLogger())
        run_server(debug=False)
``` 

then replace all the print statements with `logging.info` or similar.

Reply:
I don't understand what add_logging_handle_to_root does though